dohproxy/httpproxy.py

```diff
@@ -108,6 +113,15 @@ def on_answer(self, request, dnsr=None, dnsq=None):
 def get_app(args):
     logger = utils.configure_logger('doh-httpproxy', args.level)
     app = DOHApplication(logger=logger, debug=args.debug)
+
+    # Get trusted reverse proxies and format it for aiohttp_remotes setup
+    # trusted_reverse_proxies =
+    # [[trusted_host] for trusted_host in args.trusted]
+
+    asyncio.ensure_future(aiohttp_remotes.setup(
+        app,
+        aiohttp_remotes.XForwardedRelaxed())
```

Comment:
you should be able to test the strict method using 127.0.0.1 (until it is fixed upstream). I would say leave it as strict.
In order to provide flexibility, once this that could be done is to call `XForwardedRelaxed` if len(args.trusted) == 0 (e.g make the argument nargs='*' and provide documentation in the help text), otherwise, otherwise, call `XForwardedStrict` with the trusted list.